test/cmocka/src/math/auditory/mfcc_power_to_mel_log.m

```diff
@@ -14,14 +15,14 @@
 %
 % Copyright (c) 2022, Intel Corporation. All rights reserved.
 
-function mel_log = mfcc_power_to_mel_log(state, param, s_half)
+function mel_log = mfcc_power_to_mel_log(state, param, s_half, shift)
 
 p_lin = s_half .* conj(s_half);
 mel_log = zeros(1, param.num_mel_bins);
 for i = 1:param.num_mel_bins
 	p = sum(p_lin .* state.triangles(:, i));
 	p = max(p, param.pmin);
-	mel_log(i) = log(p) * state.log_scale;
+	mel_log(i) = (log(p) - log(2^(2 * shift))) * state.log_scale;
```

Comment:
not sure if it serves your purpose here, but you probably at least get a better precision if you do `2 * shift * log(2)`

Reply:
Probably more accurate, yes.  The reference is checked against Pytorch so I'll do this change even if it hides a bit why such equation.